convert2rhel/pkgmanager/handlers/dnf/__init__.py

```diff
@@ -120,7 +120,7 @@ def _perform_operations(self):
         for pkg in original_os_pkgs:
             # Splitting the name and arch so we can filter it out in the list
             # of packages to upgrade.
-            name, arch = tuple(pkg.rsplit("."))
+            name, arch = tuple(pkg.rsplit("."), 1)
```

Comment:
This should be
```suggestion
            name, arch = tuple(pkg.rsplit(".", 1))
```

Reply:
Thanks for the fix. Sorry for the typo, I had it working locally then messed up when I translated it to the PR.  Oops. Good catch, thanks.